### ISSUE TITLE
is_fixture_union_params throws an error when params have a type of generator

### ISSUE THREAD

**Issue by bvsn** [smarie/python-pytest-cases#115]
I have installed pytest-cases but in this test, I don't use it.

```
def iterate():
    for i in range(10):
        yield i


@pytest.mark.parametrize("idx", iterate())
    def test_fn(idx):
        pass
```
Exception when I run it:
```
platform linux -- Python 3.8.2, pytest-5.4.3, py-1.9.0, pluggy-0.13.1
rootdir: /home/bvsn/workspace/orderry/remonline, inifile: pytest.ini, testpaths: tests
plugins: cases-1.17.0, Faker-4.1.1, django-3.9.0, celery-4.4.2
collected 1 items / 1 error / 0 deselected                                                                                                        

ERROR collecting test_fn.py
../../../.local/share/virtualenvs/remonline-JzcKABx1/lib/python3.8/site-packages/pytest_cases/fixture_core1_unions.py:122: in is_fixture_union_params
    if len(params) < 1:
E   TypeError: object of type 'generator' has no len()
        params     = <generator object iterate at 0x7fa206e6a740>

```

**Comment by smarie** [comment]
Thanks for reporting ! I'll have a look

**Comment by smarie** [comment]
fixed in 2.0.4. Sorry, this was stupid :)

**Comment by bvsn** [comment]
Thank you :)
